ci/test.sh

```diff
@@ -26,13 +31,17 @@ function main {
   [[ $# -eq 0 || "$1" == '-h' || "$1" == '--help' ]] && helpAndExit 0
   REL=`dirname "$0"`
   eval "$REL/prepare_db.sh '$1'"
-  if [[ "$1" == "H2" ]]; then
-    (cd $REL/../web && npm run test)
-    mvn clean verify -q -f $REL/../lib/ -B
-    mvn clean install -q -f $REL/../rest/ -B -P history.plugin
-  elif [[ "$1" == "POSTGRES_10_4" ]]; then
+  if [[ "$1" == "H2" && "$2" == "LIB" ]]; then
+    mvn clean install -q -f $REL/../lib/ -B
+  elif [[ "$1" == "H2" && "$2" == "REST" ]]; then
+   (cd $REL/../web && npm run test)
+    mvn clean install -q -f $REL/../rest/ -B
```

Comment:
please add a comment explaining we need to package rest again because our rest documentation is build when running tests